source/tutorials.txt

```diff
@@ -8,14 +8,16 @@ Tutorials
    :titlesonly:
    :hidden:
    
+   Template App Tutorials <https://www.mongodb.com/docs/realm/tutorial/>
```

Comment:
i think this should have a more descriptive name, such as: "Atlas Device Sync Tutorials"